swagger.yaml

```diff
@@ -603,7 +606,7 @@ paths:
           description: "Unexpected Cognito response"
```

Comment:
Line 602 above still references `name` when it should reference `id`.

Current:
```
description: "Invalid group name provided"
```
Expected:
```
description: "Invalid group ID provided"
```